src/js/messaging/components/compose/MessageWrite.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ErrorableTextarea from '../../../common/components/form-elements/ErrorableTextarea';
+import { makeField } from '../../../common/model/fields';
+
+class MessageWrite extends React.Component {
+  render() {
+    const messageValue = makeField(undefined);
+
+    return (
+      <div className={this.props.cssClass}>
+        <ErrorableTextarea
+            additionalClass={`${this.props.cssClass}-text`}
```

Comment:
Is this class (I guess "messaging-write-text") being used? Also, if this were to receive multiple  or no classes, interpolation might not work as expected.